root/user/UserProfile.js

```diff
@@ -13,17 +13,30 @@ import UserAccountLayout from '../components/UserAccountLayout';
 import DescriptiveLink
   from '../static/scripts/common/components/DescriptiveLink';
 import {FLUENCY_NAMES} from '../static/scripts/common/constants';
+import compare from '../static/scripts/common/i18n';
 import commaList from '../static/scripts/common/i18n/commaList';
 import expand2react from '../static/scripts/common/i18n/expand2react';
 import bracketed, {bracketedText}
   from '../static/scripts/common/utility/bracketed';
+import * as TYPES from '../static/scripts/common/constants/editTypes';
 import escapeRegExp from '../static/scripts/common/utility/escapeRegExp';
 import commaOnlyList from '../static/scripts/common/i18n/commaOnlyList';
 import nonEmpty from '../static/scripts/common/utility/nonEmpty';
 import {formatCount, formatPercentage} from '../statistics/utilities';
 import formatUserDate from '../utility/formatUserDate';
 import {canNominate} from '../utility/voting';
 
+const ADDED_ENTITIES_TYPES = {
+  artist:    l('Artist'),
+  event:     l('Event'),
+  release:   l('Release'),
+  series:    l('Series'),
+  work:      l('Work'),
+  label:     l('Label'),
+  place:     l('Place'),
+  cover_art: l('Cover Art'),
+};
```

Comment:
* Should be localized at render time only, so use `N_l` instead.
* Plus context string `singular` is needed for proper localization of `Series`.

```suggestion
const ADDED_ENTITIES_TYPES = {
  artist:    N_l('Artist'),
  event:     N_l('Event'),
  release:   N_l('Release'),
  series:    N_l('Series', 'singular'),
  work:      N_l('Work'),
  label:     N_l('Label'),
  place:     N_l('Place'),
  cover_art: N_l('Cover Art'),
};
```